.github/workflows/ci.yml

```diff
@@ -70,9 +70,12 @@ jobs:
           # versions of our dependencies allowed by our upper bounds.
```

Comment:
This approach of only testing the oldest and newest supported versions makes sense for most projects, but since this particular project is directly importing the GHC API and that API changes a lot between releases, I think we should test every major GHC version we support.

Reply:
And then let's add a "tested-with" line to the cabal file listing those same GHC versions

Reply:
fixed in 9385576a99642cb60e445451e9ef152110a80a24